main.go

```diff
@@ -18,9 +18,16 @@ const LOGGER_TYPE_LOG = "log"
 const LOGGER_TYPE_ERROR = "error"
 
 type Context struct {
-	Logger Logger
-	Req    Request
-	Res    Response
+	logger Logger
+
+	Req ContextRequest
+	Res ContextResponse
+}
+
+func NewContext(logger Logger) Context {
+	return Context{
+		logger: logger,
+	}
```

Comment:
```suggestion
	return Context {
		logger: logger,
	}
```
Formatting seems off. Please run it through a formatter

Reply:
Formatter for Go runs on every save. I tried to add space, but it removed it.
This is the way to define new structs. It's same as

```
[]int{ 1,2,3 }
interface{}{ User: nil }
```